File: src/main/java/io/github/chaosdave34/kitpvp/items/impl/tank/TankAxe.java
package io.github.chaosdave34.kitpvp.items.impl.tank;

import io.github.chaosdave34.kitpvp.abilities.Ability;
import io.github.chaosdave34.kitpvp.abilities.AbilityHandler;
import io.github.chaosdave34.kitpvp.items.CustomItem;
import net.kyori.adventure.text.Component;
import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public class TankAxe extends CustomItem {
    public TankAxe() {
        super(Material.IRON_AXE, "tank_axe");
    }

    @Override
    public @NotNull Component getName() {
        return createSimpleItemName("Tank Axe");
    }

    @Override
    public @NotNull List<Ability> getAbilities() {
        return List.of(AbilityHandler.FORTIFY);
    }

    @Override
    protected void additionalModifications(ItemStack itemStack) {
        setCustomModelData(itemStack, 1);
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/items/impl/tank/TankAxe.kt
package io.github.chaosdave34.kitpvp.items.impl.tank

import io.github.chaosdave34.kitpvp.abilities.Ability
import io.github.chaosdave34.kitpvp.abilities.AbilityHandler
import io.github.chaosdave34.kitpvp.items.CustomItem
import net.kyori.adventure.text.Component
import org.bukkit.Material
import org.bukkit.inventory.ItemStack

class TankAxe : CustomItem(Material.IRON_AXE, "tank_axe") {

    override fun getName(): Component = createSimpleItemName("Tank Axe")

    override fun getAbilities(): List<Ability> = listOf(AbilityHandler.TORNADO)

    override fun additionalModifications(itemStack: ItemStack) {
        itemStack.setCustomModelData(1)
    }
}